docs/en/reference/information_schema/tables.md

```diff
@@ -0,0 +1,33 @@
+---
+displayed_sidebar: "English"
+---
+
+# tables
+
+`tables` provides information about tables.
+
+The following fields are provided in `tables`:
+
+| **Field**       | **Description**                                              |
+| --------------- | ------------------------------------------------------------ |
+| TABLE_CATALOG   | Name of the catalog that stores the table.                   |
+| TABLE_SCHEMA    | Name of the database that stores the table.                  |
+| TABLE_NAME      | Name of the table.                                           |
+| TABLE_TYPE      | Type of the table. Valid values: "BASE TABLE" or "VIEW".     |
+| ENGINE          | Engine type of the table. Valid values: "StarRocks", "MySQL", "MEMORY" or an empty string. |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/690fd430-b513-4902-a05a-73325344e637)
wrap these in code format, not in quotation marks